independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/impl/multipart/QuarkusMultipartFormDataPart.java

```diff
@@ -133,6 +160,10 @@ public Buffer content() {
         return content;
     }
 
+    public Multi<Byte> multiByteContent() {
```

Comment:
Something we could improve (later) is to also support Multi<byte[]>, because it's what you receive most of the time (chunks, not bytes one by one).